crates/iota-indexer/src/models/transactions.rs

```diff
@@ -316,8 +332,8 @@ impl StoredTransaction {
             events,
             object_changes,
             balance_changes,
-            timestamp_ms: Some(self.timestamp_ms as u64),
-            checkpoint: Some(self.checkpoint_sequence_number as u64),
+            timestamp_ms,
```

Comment:
Nit: You could use [then_some](https://doc.rust-lang.org/std/primitive.bool.html#method.then_some) here directly. E.g.

```rust
timestamp_ms: self.is_checkpointed_transaction.then_some(self.timestamp_ms as u64),
```

Reply:
I couldn't do it at this point because 'self' is partially borrowed at this point.

I used the 'then_some' construction earlier on in the function.